pulsar-sql/presto-pulsar/src/main/java/org/apache/pulsar/sql/presto/PulsarSqlSchemaInfoProvider.java

```diff
@@ -98,6 +99,7 @@ public String getTopicName() {
     }
 
     private SchemaInfo loadSchema(BytesSchemaVersion bytesSchemaVersion) throws PulsarAdminException {
+        Thread.currentThread().setContextClassLoader(InjectionManagerFactory.class.getClassLoader());
```

Comment:
if you set the Thread Context Classloader you have to restore it in a finally block

Reply:
Thanks for reminding me, I'll fix it.